config/initializers/new_framework_defaults.rb

```diff
@@ -20,6 +20,3 @@
 
 # Require `belongs_to` associations by default. Previous versions had false.
 Rails.application.config.active_record.belongs_to_required_by_default = false
-
-# Do not halt callback chains when a callback returns false. Previous versions had true.
-ActiveSupport.halt_callback_chains_on_return_false = true
```

Comment:
Will this break any expectations within the app that returning false within a callback halts the chain?

Reply:
Thanks for bringing this up. I dug into all of our existing callbacks and they should not be affected by this change.

For current before callbacks, the methods either end with a statement that will always be true (a var assignment of a truthy value, a string, etc), they explicitly have `true` as the final return value to ensure the callback won't be halted or they raise an exception which intentionally halts the callbacks.